photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/USBCameraSource.java

```diff
@@ -38,7 +38,7 @@ public class USBCameraSource extends VisionSource {
     private final Logger logger;
     private final UsbCamera camera;
     protected GenericUSBCameraSettables settables;
-    protected FrameProvider usbFrameProvider;
+    protected USBFrameProvider usbFrameProvider;
```

Comment:
This is causing CI to fail, why change?

Reply:
Because I need access to a member function only available on USBFrameProvider, coz i'm lazy. I should do this the not lazy way, though.

Reply:
did it the smarter way